packages/cli/src/generator.test.ts

```diff
@@ -139,7 +139,26 @@ export interface IGetNotificationsQuery {
         types,
         {} as ParsedConfig,
       );
-      expect(result).toMatchSnapshot();
```

Comment:
I would prefer a snapshot instead, any reasons why a string would be better?

Reply:
It simply matches the style of the other tests, they could all be snapshots or all be inline strings, or even be inline snapshots. This change was just one for consistency to have less surprises due to differing conventions when reading the code.

To be honest I'd prefer if they were all snapshots (easier to update if they change).

Reply:
Makes sense, lets migrate them all to snapshots later.